### PR TITLE
fix: overwrite stream for ChatOpenAI in runtime

### PR DESCRIPTION
**Pull request by viccao95**
@hwchase17, @baskaryan

### CODE REVIEW

File: libs/langchain/langchain/chat_models/openai.py [hunk 1]
```diff
@@ -320,9 +320,8 @@ def _generate(
         run_manager: Optional[CallbackManagerForLLMRun] = None,
         **kwargs: Any,
     ) -> ChatResult:
-        message_dicts, params = self._create_message_dicts(messages, stop)
-        params = {**params, **kwargs}
-        if self.streaming:
```
Comment (baskaryan) [inline libs/langchain/langchain/chat_models/openai.py:325]: wait why are we ignoring self.streaming

Comment (ifplusor) [reply libs/langchain/langchain/chat_models/openai.py:325]: > wait why are we ignoring self.streaming

`self.streaming` has be used to initialize `params["stream"]` in `self._default_params`.

`self._create_message_dicts` -> `self._client_params` -> `self._default_params`

Comment (baskaryan) [reply libs/langchain/langchain/chat_models/openai.py:325]: ah good call! should we del L327 then?

Comment (ifplusor) [reply libs/langchain/langchain/chat_models/openai.py:325]: Maybe it's useless, except making sure the stream is a boolean.

**Review by nfcampos — APPROVED** [review]

**Pull request merged**